### DISCUSSION TITLE
Higher order tasks

### DISCUSSION THREAD

**Discussion by codito** [focustimerhq/FocusTimer#539] [General]
I took a quick look at the issues backlog, following themes showed up. Goal is to give us a clear sense of - a) issues/bugs in current app, b) feature requests/vision, c) tasks blocking adoption/community involvement.

1. Probably "the" one issue that stands out is system freeze: https://github.com/gnome-pomodoro/gnome-pomodoro/issues?q=is%3Aissue+is%3Aopen+freeze
2. Next one is installation/build in various distros: https://github.com/gnome-pomodoro/gnome-pomodoro/issues?q=is%3Aissue+is%3Aopen+build+ and attempts to modernize the build with `meson`
3. We've a steady stream of feature requests and enhancements around extending etc. How can we be a platform i.e. enable the community to hack/contribute?

Will spend some more time triaging/categorizing them later this week. Buckets (2) and (3) can be solved by improving our tooling, diagnostics (logging), and documentation. Any other top of mind tasks we should prioritize? cc: @gnome-pomodoro/devs

**Comment by codito** [comment]
As part of improving tooling, we will create issue templates to differentiate between features, bugs and general discussion.

**Reply by josehbez** [reply]
I suggest  using these templates, What do you think @gnome-pomodoro/devs ?


### Bug Report 
![screenshot-github com-2021 01 25-17_24_28](https://user-images.githubusercontent.com/13144779/105778567-3eb53a80-5f32-11eb-9ffa-537ef79d979d.png)


### Feature
![screenshot-github com-2021 01 25-17_23_56](https://user-images.githubusercontent.com/13144779/105778497-22190280-5f32-11eb-8354-f8193f9245ed.png)


### Question 
![screenshot-github com-2021 01 25-17_26_04](https://user-images.githubusercontent.com/13144779/105778670-6dcbac00-5f32-11eb-8f76-b9c91121726e.png)


### Pull request 
![screenshot-github com-2021 01 25-17_26_35](https://user-images.githubusercontent.com/13144779/105778718-88058a00-5f32-11eb-945c-9db23539fa04.png)

**Reply by josehbez** [reply]
PR https://github.com/gnome-pomodoro/gnome-pomodoro/pull/546

**Reply by kamilprusko** [reply]
Merged :heart:

As to Disqus - it's not that great for community support. I'll look for replacements.

**Reply by josehbez** [reply]
> Merged 
> 
> As to Disqus - it's not that great for community support. I'll look for replacements.

Mayb: 

- Discord 
- Matrix 
- Telegram
- Slack

@kamilprusko

**Comment by kamilprusko** [comment]
I would add to that list port to Gtk 4.0. It's coming this March.

> Installation/build in various distros

We could use Flatpak, but the extension will need to be installed separately. Switching from Vala to gjs would simplify code and reduce number of build dependencies.

**Reply by josehbez** [reply]
+1 Flatpak

+1 extension separate

**Comment by josehbez** [comment]
We need a simple roadmap-   Why ?

* List issues to fix (e.g. freeze X distro)
* New improvements to approved (e.g. new tech to builds)
* New features to coming  (e.g. gnome-pomodoro stats )

From gnome-pomodoro, gnome-pomodoro-website and gnome-pomodoro-integrations

So that  any contributor can claim to fix or create.
